src/internal/createSharpOptions.js

```diff
@@ -12,9 +14,9 @@ import type {ImageOptions, SharpOptions} from '../types';
  */
 const createSharpOptions = (options: ImageOptions, meta: *): SharpOptions => {
   const result = {};
-  if (typeof options.format === 'string') {
-    result.toFormat = options.format;
-  }
+
+  const [format, formatOptions] = parseFormat(options.format);
```

Comment:
Add `if (options.format)` guard? Probably cases where one just wants a pass-thru.

Reply:
When is `options.format` ever allowed to be undefined?

Reply:
`format` is not optional on the `ImageObject` interface (https://github.com/metalabdesign/sharp-loader/blob/master/src/types.js#L23)

Reply:
Hum. I think I had always thought of it possibly being nullable in the case one wanted a pass-thru (which the old code supported). I mean, I'm fine with this, it's in some way a breaking change, not that enough people use this library that it would be a big deal.

If you want to keep as is I'll put out a new major, if not then it's a one-line fix.

Reply:
What is the behavior if format is not defined? Will it infer from the type of loaded file?

Reply:
Yeah, it should just not change the type. This was my understanding of `sharp`'s behaviour and this effectively is what was happening with your issue before – generating all those PNGs instead of jpegs was because `toFormat` was left undefined.

Reply:
Updated. See #0f7e316